src/steps/02-EvaluationCriteria/JandA/MinimumRequirements.vue

```diff
@@ -1,25 +1,188 @@
 <template>
-  <v-form ref="form" lazy-validation>
-    <v-container fluid class="container-max-width">
-      <v-row>
-        <v-col class="col-12">
-          <h1 class="mb-3">
-            Tell us about your minimum government requirements
-          </h1>
-          <p>
-            Intro paragraph
-          </p>
-        </v-col>
-      </v-row>
-    </v-container>    
-  </v-form>
+  <div>
+    <v-form ref="form" lazy-validation>
+      <v-container fluid class="container-max-width">
+        <v-row>
+          <v-col class="col-12">
+            <h1 class="mb-3">
+              Tell us about your minimum government requirements
+            </h1>
+            <p>
+              In the field below, explain what is being supported and provide any
+              information regarding your verified minimum requirements. Discuss, if
+              applicable, how the required delivery/performance date impacted the
+              decision to restrict competition. You can fill in the blank to complete
+              the suggested sentences or write your own explanation.
+            </p>
+            <ATATTextArea
+                id="minGovReqExplanation"
+                class="max-width-740"
+                :rows="7"
+                :rules="
+              [
+                $validators.required(
+                  'Enter your minimum government requirements.'
+                ),
+                $validators.maxLength(
+                  1000,
+                  'Please limit your description to 1000 characters or less'
+                ),
+              ]
```

Comment:
AC 3.c.i -- Validation should fire if user clicks in then out without making changes to default copy.
AC 6.b.i.1 -- Validation isn't firing when clicking Continue button if NO changes to default copy.